application/src/main/java/org/togetherjava/tjbot/features/utils/RateLimiter.java

```diff
@@ -0,0 +1,60 @@
+package org.togetherjava.tjbot.features.utils;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Custom rate limiter.
+ */
+public class RateLimiter {
```

Comment:
worthless javaodc. please improve. especially because this is a general purpose utility (judging by the package)

Reply:
Improved, tell me what you think.